docs/operate/reference/advanced-modules/_index.md

```diff
@@ -46,6 +46,74 @@ Running {{< glossary_tooltip term_id="modular-resource" text="modular resources"
 
 However, if you are unable to use modular resources because you need to host `viam-server` on a non-Linux system or have an issue with compilation, you may need to [implement a custom component and register it on a server configured as a remote](/operate/reference/advanced-modules/custom-components-remotes/) on your machine.
 
+## Package and deploy using Docker
+
+In rare cases, you may need to package and deploy a module using Docker.
+Use cases for this include:
+
+- Your module has complex system dependencies that cannot be installed on a machine.
+- You have a large bundle and want to use layer caching to reduce the size of the download.
+- You have specific security requirements that are difficult to meet with the default module deployment.
+
+If you choose to deploy your module using Docker, we recommend creating a first run script or binary to run any necessary setup steps.
```

Comment:
I'd like to better indicate that "first run" is a single thing, so no one mistakes it for, say, the first of many different run scripts or something. Would putting quotes around "first run" be appropriate? What about an underscore in first_run? I dunno, maybe leave as-is if that's standard in the docs.

Reply:
```suggestion
If you choose to deploy your module using Docker, we recommend creating a "first run" script or binary to run any necessary setup steps.
```